ct/proxmox-datacenter-manager.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/raw/main/misc/build.func)
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: CrazyWolf13
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: Proxmox Server Solution GmbH
+
+# App Default Values
+APP="proxmox-datacenter-manager"
+var_tags="backup"
+var_cpu="2"
+var_ram="2048"
+var_disk="10"
+var_os="debian"
+var_version="12"
+var_unprivileged="1"
+
+# App Output & Base Settings
+header_info "$APP"
+base_settings
+
+# Core
+variables
+color
+catch_errors
+
+function update_script() {
+    header_info
+    check_container_storage
+    check_container_resources
+    if [[ ! -d /var ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
```

Comment:
Can you check where it actually gets installed and put the path here.

Reply:
Sure, I tought as this is just an apt package, and tteck did it like this in pbs, I tought I'll stick to it, however sure, just added a path to the proxmox-datacenter-manager admin binary.